lib/platforms.nix

```diff
@@ -8,8 +8,9 @@ rec {
   openbsd = ["i686-openbsd" "x86_64-openbsd"];
   netbsd = ["i686-netbsd" "x86_64-netbsd"];
   cygwin = ["i686-cygwin" "x86_64-cygwin"];
-  unix = linux ++ darwin ++ freebsd ++ openbsd;
-  all = linux ++ darwin ++ cygwin ++ freebsd ++ openbsd;
+  illumos = ["x86_64-solaris"];
```

Comment:
Nitpick: why use two different names for the (same) nix platform (illumos and solaris)?